Description: Fix classpath using /usr/share/java instead of lib/
Author: Damien Raude-Morvan <drazzib@debian.org>
Author: gil cattaneo <puntogil@libero.it>
Origin: http://pkgs.fedoraproject.org/cgit/classycle.git/tree/
Last-Updated: 2012-09-03
--- a/build.xml
+++ b/build.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<project default="jar" name="Classycle" basedir=".">
+<project default="distribution" name="Classycle" basedir=".">
   <property name="version" value="1.4.2"/>
 
   <!-- prefix that allows to reference environment variables ===============-->
@@ -19,8 +19,8 @@
 
   <!-- set classpath for compilation of src ================================-->
   <path id="src_classpath">
-    <fileset dir="lib">
-      <include name="**/*.jar"/>
+    <fileset dir="/usr/share/java">
+      <include name="ant-testutil.jar"/>
     </fileset>
   </path>
 
@@ -94,12 +94,11 @@
               version="true"
               use="true"
               windowtitle="Classycle"
-              overview="overview.html"
               failonerror="true"/>
   </target>
 
   <!-- generate distribution ===============================================-->
-  <target name="distribution" depends="jar" 
+  <target name="distribution" depends="jar,apidoc"
           description="generate a distribution">
     <delete file="src.zip ${DIST_NAME}"/>
     <zip destfile="src.zip" basedir="src"/>
